### ISSUE TITLE
Debugging terminal, the directory is not empty

### ISSUE THREAD

**Issue by firebrand76** [bleachbit/bleachbit#1669]
![Screenshot 2024-05-07 120334](https://github.com/bleachbit/bleachbit/assets/169153860/a3716e41-19b3-4c0b-8429-71fc4d91ac58)
I am not able to delete this file shown in the screenshot. The debugging terminal states 'the directory is not empty' It is fs capricorn\temp etc. Should I be worried? If so, what can I do about it? Thanks.

**Comment by az0** [comment]
Thanks for your patience, as I am working through the backlog.

tl;dr: No need to worry

Based on a quick search, it is normal for F-Secure Internet Security to make this directory, so that's fine.

As a general statement about BleachBit, you may see this warning "directory is not empty" when there is a file in a directory that cannot be deleted. Often, this is caused by a locked file inside that directory. F-Secure probably has the file locked/open, so BleachBit cannot delete it now. 

When you run BleachBit in "clean" mode, you may see "marked for deletion" instead of "delete". This is easiest to spot by running "clean" twice in a row because it shows up both times, but the second time, the files that were deleted normally are no longer there.
